Frontend/package.json

```diff
@@ -21,8 +21,7 @@
     "lodash": "^4.17.21",
     "vue": "^3.2.21",
     "vue-router": "^4.0.12",
-    "vue3-datepicker": "^0.2.5",
-    "vuex": "^4.0.2"
```

Comment:
Vuex wird immer noch in folgenden Files benötigt:
* FlightDetails
* CommentEditor
* Comment
* CommentReply

Hab es versucht Stück für Stück auszukommentieren. Aber ich tappe da immer wieder in den nächsten Error.

Reply:
Merkwürdig, dachte ich hätte alles entfernt, da muss ich noch mal ran.